components/Hero/src/index.scss

```diff
@@ -147,6 +147,95 @@
   padding-block-end: var(--denhaag-hero-theme-page-title-padding-block-end);
 }
 
+/* added .denhaag-hero to --them-image, otherwise need to use !important to overwrite */
```

Comment:
We gebruiken CSS variabelen. 

`--denhaag-hero-background-color: var(--denhaag-hero-theme-image-background-color);` zou niet voor een important moeten zorgen. CSS variabele zijn gemaakt om overschreven te worden.